numpy/core/src/multiarray/compiled_base.c

```diff
@@ -278,7 +278,7 @@ arr_histogramdd_uniform(PyObject *NPY_UNUSED(self), PyObject *args, PyObject *kw
         PyErr_SetString(PyExc_TypeError, "Couldn't parse the input edges.");
         goto fail;
     }
-    npy_intp nedges_total = 0;
```

Comment:
This was fine. Appveyer's error message was misleading. Travis-CI caught the actual bug which was fixed in commit cb117ec194a9073e4cbfa78800fb7d6edd8bb339